x-pack/plugins/serverless_security/public/components/get_started/types.ts

```diff
@@ -56,8 +56,10 @@ export interface Card {
   stepsLeft?: number;
 }
 
+export type ActiveCards = Record<SectionId, Record<CardId, ActiveCard>>;
+
 export enum ProductId {
-  analytics = 'analytics',
+  security = 'security',
```

Comment:
Product type security displayed as Analytics in toggle but it's product id changed to security to aligned with product types' configs

Reply:
The schema and types for the ProductLines are already defined at:
https://github.com/elastic/kibana/blob/a8e07e87091981f191fd2547261ccff1e2588709/x-pack/plugins/serverless_security/common/config.ts#L10-L15

We are duplicating the definition here. It would be better to define the enum there and set the schema using the enum, so we define them only once:

```
export enum ProductLine {
  security = 'security',
  cloud = 'cloud',
  endpoint = 'endpoint',
}

export const productLine = schema.oneOf([
  schema.literal(ProductLine.security),
  schema.literal(ProductLine.endpoint),
  schema.literal(ProductLine.cloud),
]);
```